crates/router/src/connector/itaubank.rs

```diff
@@ -170,9 +302,12 @@ impl ConnectorIntegration<api::Authorize, types::PaymentsAuthorizeData, types::P
     fn get_url(
         &self,
         _req: &types::PaymentsAuthorizeRouterData,
-        _connectors: &settings::Connectors,
+        connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Err(errors::ConnectorError::NotImplemented("get_url method".to_string()).into())
+        Ok(format!(
+            "{}itau-ep9-gtw-pix-recebimentos-ext-v2/v2/cob",
```

Comment:
Hard-coding this part of the URL can require us significant build time if and when we are required to update this URL. Can we have this part of the URL being read from configs?